### PR TITLE
log prints

### PR DESCRIPTION
**Pull request by kmassada**

### CODE REVIEW

File: pubsub/pullstream.go [hunk 1]
```diff
@@ -47,6 +48,7 @@ func newPullStream(ctx context.Context, streamingPull streamingPullFunc, subName
 			spc, err := streamingPull(ctx, gax.WithGRPCOptions(grpc.MaxCallRecvMsgSize(maxSendRecvBytes)))
 			if err == nil {
 				recordStat(ctx, StreamRequestCount, 1)
+				log.Print("starting open")
 				err = spc.Send(&pb.StreamingPullRequest{
 					Subscription: subName,
 					// We modack messages when we receive them, so this value doesn't matter too much.
```
Comment (jba) [inline pubsub/pullstream.go]: Immediately after the spc.Send call, add

`log.Printf("open returned %v", err)`

File: pubsub/pullstream.go [hunk 2]
```diff
@@ -160,6 +162,7 @@ func (s *pullStream) Send(req *pb.StreamingPullRequest) error {
 		recordStat(s.ctx, NackCount, int64(zeroes))
 		recordStat(s.ctx, ModAckCount, int64(len(req.ModifyDeadlineSeconds)-zeroes))
 		recordStat(s.ctx, StreamRequestCount, 1)
+		log.Print("Send Complete")
```
Comment (jba) [inline pubsub/pullstream.go]: Actually, the Send hasn't started yet. This code should be
```
log.Print("starting Send")
err := spc.Send(req)
log.Printf("Send returned %v", err)
return err
```

File: pubsub/pullstream.go [hunk 3]
```diff
@@ -169,6 +172,7 @@ func (s *pullStream) Recv() (*pb.StreamingPullResponse, error) {
 	err := s.call(func(spc pb.Subscriber_StreamingPullClient) error {
 		var err error
 		recordStat(s.ctx, StreamResponseCount, 1)
+		log.Print("starting send")
```
Comment (jba) [inline pubsub/pullstream.go]:
```
log.Print("starting Recv")
res, err = spc.Recv()
log.Printf("Recv returned %v", err)
```

File: pubsub/pullstream.go [hunk 4]
```diff
@@ -47,12 +48,14 @@ func newPullStream(ctx context.Context, streamingPull streamingPullFunc, subName
...
 			if err == nil {
 				recordStat(ctx, StreamRequestCount, 1)
+				log.Print("starting open")
 				err = spc.Send(&pb.StreamingPullRequest{
 					Subscription: subName,
 					// We modack messages when we receive them, so this value doesn't matter too much.
 					StreamAckDeadlineSeconds: 60,
 				})
 			}
+			log.Printf("open returned %v", err)
```
Comment (jba) [inline pubsub/pullstream.go]: This isn't immediately after the Send call. It is after the if statement. Move this up one line.

**Comment by jadekler** [comment]
Hello! Closing this PR as I believe the original reason for opening it (debugging an issue) has been resolved. Let me know if that's not the case and we can re-open.

**Pull request closed without merging**